src/active-response/wazuh-slack.c

```diff
@@ -0,0 +1,265 @@
+/* Copyright (C) 2015-2021, Wazuh Inc.
+ * All right reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "active_responses.h"
+
+/**
+ * Get json with the data to share on slack from an alert. Example:
+ *
+ * @param alert Alert to extract info
+ * @return  json object
+ * */
+static cJSON *format_output(cJSON *alert);
+
+int main (int argc, char **argv) {
+    (void)argc;
+    CURL *curl;
+    CURLcode res;
+    struct curl_slist *headers = NULL;
+    char errbuf[CURL_ERROR_SIZE];
+    char input[BUFFERSIZE];
+    char *site_url;
+    char *action;
+    cJSON *alert_json = NULL;
+    cJSON *input_json = NULL;
+    cJSON *output_json = NULL;
+
+    write_debug_file(argv[0], "Starting");
+
+    memset(input, '\0', BUFFERSIZE);
+    if (fgets(input, BUFFERSIZE, stdin) == NULL) {
+        write_debug_file(argv[0], "Cannot read input from stdin");
+        return OS_INVALID;
+    }
+
+    write_debug_file(argv[0], input);
+
+    input_json = get_json_from_input(input);
+    if (!input_json) {
+        write_debug_file(argv[0], "Invalid input format");
+        return OS_INVALID;
+    }
+
+    action = get_command(input_json);
+    if (!action) {
+        write_debug_file(argv[0], "Cannot read 'command' from json");
+        cJSON_Delete(input_json);
+        return OS_INVALID;
+    }
+
+    if (strcmp("add", action)) {
```

Comment:
Since `add` and `delete` commands are both valid, I will do this like it is done in the file `restart-wazuh.c`. Take a look at that file.